external/kafka/src/main/scala/org/apache/gearpump/streaming/kafka/lib/KafkaUtil.scala

```diff
@@ -87,6 +87,17 @@ object KafkaUtil {
     }
   }
 
+  def deleteTopic(zkClient: ZkClient, topic: String): Unit = {
+    try {
+      AdminUtils.deleteTopic(zkClient, topic)
+    } catch {
+      case e: Exception =>
+        LOG.error(e.getMessage)
+    } finally {
+      zkClient.close()
```

Comment:
Good point but I don't want `KafkaStorage` to worry about the zookeeper connection, whose creation and close should be contained in this method. Now I passed in a connect function so there will be no `ZkClient` objects in `KafkaStorage`. I also applied this to other methods that interact with zookeeper. The reason I don't just create a `ZkClient` here is to make the method testable.